core/src/main/scala/kafka/log/Log.scala

```diff
@@ -292,6 +305,20 @@ class Log(val dir: File,
    */
   def numberOfSegments: Int = segments.size
 
+
+  /**
+   * Returns compiled information about segments
+   */
+  def segmentsInfo: util.ArrayList[String] = {
```

Comment:
Given that we already expose the segments via `logSegments`, is this method actually needed?